<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="659" started-at="2019-08-09T07:31:42Z" finished-at="2019-08-09T07:31:42Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:casestudy.runner@77cd7a0]" name="feature" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="659" started-at="2019-08-09T07:31:42Z" finished-at="2019-08-09T07:31:42Z">
      <class name="casestudy.runner">
        <test-method status="FAIL" signature="setUpClass()[pri:0, instance:casestudy.runner@77cd7a0]" name="setUpClass" is-config="true" duration-ms="0" started-at="2019-08-09T13:01:42Z" finished-at="2019-08-09T13:01:42Z">
          <exception class="cucumber.runtime.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in casestudy.registrationstepdefinition.navigateToHomePage() in file:/C:/week4/case2/target/classes/ and casestudy.login.navigateToHomePage() in file:/C:/week4/case2/target/classes/]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.DuplicateStepDefinitionException: Duplicate step definitions in casestudy.registrationstepdefinition.navigateToHomePage() in file:/C:/week4/case2/target/classes/ and casestudy.login.navigateToHomePage() in file:/C:/week4/case2/target/classes/
	at cucumber.runtime.RuntimeGlue.addStepDefinition(RuntimeGlue.java:33)
	at cucumber.runtime.java.JavaBackend.addStepDefinition(JavaBackend.java:117)
	at cucumber.runtime.java.MethodScanner.scan(MethodScanner.java:68)
	at cucumber.runtime.java.MethodScanner.scan(MethodScanner.java:41)
	at cucumber.runtime.java.JavaBackend.loadGlue(JavaBackend.java:78)
	at cucumber.runtime.Runtime.<init>(Runtime.java:91)
	at cucumber.runtime.Runtime.<init>(Runtime.java:69)
	at cucumber.runtime.Runtime.<init>(Runtime.java:65)
	at cucumber.api.testng.TestNGCucumberRunner.<init>(TestNGCucumberRunner.java:42)
	at cucumber.api.testng.AbstractTestNGCucumberTests.setUpClass(AbstractTestNGCucumberTests.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="SKIP" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:casestudy.runner@77cd7a0]" name="feature" duration-ms="0" started-at="2019-08-09T13:01:42Z" description="Runs Cucumber Feature" data-provider="features" finished-at="2019-08-09T13:01:42Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:441)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1300)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1015)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1104)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: java.lang.NullPointerException
	at cucumber.api.testng.AbstractTestNGCucumberTests.features(AbstractTestNGCucumberTests.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="SKIP" signature="tearDownClass()[pri:0, instance:casestudy.runner@77cd7a0]" name="tearDownClass" is-config="true" duration-ms="0" started-at="2019-08-09T13:01:42Z" finished-at="2019-08-09T13:01:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- casestudy.runner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
